app/src/main/java/org/fossasia/openevent/app/core/event/list/EventListFragment.java

```diff
@@ -266,8 +183,11 @@ public void changeToNormalMode() {
     }
 
     @Override
-    public void resetEventsList() {
-        eventListAdapter.categorizeEvents();
-        binding.bottomNavigation.setSelectedItemId(R.id.action_live);
+    public void setEventsList(List<Event> events) {
+        this.events = events;
+        getPresenter().setEvents(events);
```

Comment:
`getPresenter()` is coming out to be null when called by ViewPager's `getItem()`, thus resulting in NPE